Controller/ProjectController.java

```diff
@@ -228,7 +238,7 @@ public void actionPerformed(ActionEvent e){
                 filteredProjectList.add(newProject);
                 projectList.saveProjectCountIncrement();
                 addNewProjectToTable(newProject);
-                projectView.setupLecturerAddProjectPanel();
+                projectView.setupAdminAddProjectPanel();
```

Comment:
err what is this, why u do this

Reply:
so i remove line 41 and add back projectView.setupLecturerAddProjectPanel();   to line 31. P.S. even though i didnt touch it b4